### PR TITLE
85 contribution guide

### PR DESCRIPTION
**Pull request by kazewong**
Add contribution guide about new features, tutorial, and support information to the contribution guide.

Also link the contribution guide in the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 
 ### Expectations
 
-flowMC is developed and maintained in my spare time and, while I try to be responsive, I don't always get to every issue immediately.
-If it has been more than a week or two, feel free to ping me to try to get my attention.
-This is subject to changes as the community grows
+flowMC is developed and maintained in my spare time and, while I try to be
+responsive, I don't always get to every issue immediately. If it has been more
+than a week or two, feel free to ping me to try to get my attention. This is
```
Comment (marylou-gabrie) [inline CONTRIBUTING.md:6]: By ping me, do you mean quoting you on the issue thread?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,3 +16,38 @@ and clear description**, as much relevant information as possible, and the
...
+Open a new GitHub pull request with the example or tutorial. The example should
+be self-contained and keep import from other packages to minimal. Leave the
+case-specific analysis detail out. For more extensive tutorial, we encourage the
+community to link the minimal example hosted on the flowMC documentation to
+documentation from other packages.
+
+### Do you have question about the code?
+
+Do not open an issue. Instead, find whether there are already existing threads
+on the [flowMC support group](https://groups.google.com/u/1/g/flowmc). If not,
```
Comment (marylou-gabrie) [inline CONTRIBUTING.md:52]: We should add instructions on how to join the google group. Currently, I have an error "unavailable content" when following the link.

**Review by marylou-gabrie — COMMENTED** [review]
I suggested a couple of adjustments through comments.

File: CONTRIBUTING.md (hunk 2 above)
Comment (marylou-gabrie) [reply CONTRIBUTING.md:52]: This has been fixed

**Review by marylou-gabrie — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kazewong) [reply CONTRIBUTING.md]: Done.

**Pull request merged**